SparkleSpin/Controllers/PlayerViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  PlayerViewController.swift
+//  SparkleSpin
+//
+//  Created by Britney Smith on 3/16/19.
+//  Copyright © 2019 Britney Smith. All rights reserved.
+//
+
+import UIKit
+
+class PlayerViewController: UIViewController {
+    
+    @IBOutlet weak var playerTableView: UITableView!
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setNavigationBarTitle()
+        registerNib()
+        setupTableView()
+    
+    }
+    
+    private func setNavigationBarTitle() {
+        navigationItem.title = "Choose Players!"
+    }
+    
+    private func setupTableView() {
+        playerTableView.dataSource = self
+        playerTableView.delegate = self
+        playerTableView.backgroundColor = ThemeColor.Light.primaryColor
+        playerTableView.separatorStyle = .none
+        playerTableView.rowHeight = UITableView.automaticDimension
+        playerTableView.estimatedRowHeight = 200
```

Comment:
Might be cool to put numbers behind constants in a file similar to what you did for `SegueID`, `NibID ` and `CellID`

Reply:
Good idea!

Reply:
@tolkiana Changed!